### DISCUSSION TITLE
无法添加闪电字

### DISCUSSION THREAD

**Discussion by feyzhen** [rimeinn/rime-moran#116] [Q&A] [answered]
我想将刚刚 gg，嘿嘿 hh，宝宝 bb，了 le，我嘞个逗 wlgd等词作为闪电词加入词库中。我下载的是简体全包，部署后分别尝试了将词加在moran_fixed_simp.dict.yaml和moran_fixed.dict.yaml以及moran.extended.dict.yaml中，重新部署后仍然选不出来。除非是添加在moran_custom_phrases.txt里面，会作为首个候选。
![1](https://github.com/user-attachments/assets/bf394b2d-8af2-4f55-9c90-a1080d4d10b7)
![2](https://github.com/user-attachments/assets/60de7b65-7565-4c46-a56a-2e286995dd5f)
![3](https://github.com/user-attachments/assets/403657e8-a391-46b1-9426-7e79a2a91b8f)
![4](https://github.com/user-attachments/assets/ee7eca45-90e8-4390-baed-d1c35c6580fc)

**Comment by ksqsf** [comment]
从截图上看，感觉你是不是用了空格分隔？词库中应该使用tab键分隔。（如果你不方便打 tab 符号，可以用魔然内置的 /tab 输入）

**Reply by feyzhen** [reply]
可能是我sublime里面的tab宽度设成2的原因吗。但是我用查找替换功能看得出来是tab。
![图片](https://github.com/user-attachments/assets/07b749f8-e6eb-407b-8a88-0bbce0be4bf3)

**Reply by ksqsf** [reply]
那大概是因为某些原因词典部署失败了，只看截图感觉没啥问题。看一下错误日志？（搜索 dict 相关错误）

**Reply by feyzhen** [reply]
好像词库都导入失败了
![图片](https://github.com/user-attachments/assets/35d1582c-ff63-45a4-9812-bbe1e9aafe73)
![图片](https://github.com/user-attachments/assets/dd1947af-05f6-456f-a63f-27acd5c4e6d9)
![图片](https://github.com/user-attachments/assets/efcd6122-07db-4bbb-a446-70fb65c6558f)

**Reply by ksqsf** [reply]
确实是词库部署失败了。可否向上看看日志，有没有提到具体是什么错误？

**Reply by feyzhen** [reply]
![图片](https://github.com/user-attachments/assets/255328df-0cbb-4565-bac2-95d5645ef89e)
要不我试一下重新下载包然后部署一遍

**Reply by ksqsf** [reply]
报错提到moran fixed simp第11行有语法错误（其他几个文件均是类似错误）。这可能是之前修改时把语法改错了，也可能是编辑器自动处理时改错了。我估计把 --- 和 ... 之间的内容恢复为原始内容即可修复。

**Comment by feyzhen** [comment]
我将文件格式核对修改之后没报错了，然后设置的闪电字还是没有，闪电图标也不见了。😣。

**Reply by ksqsf** [reply]
建议删除build目录后再重新部署试试。然后您可以尝试切换到「魔然·字词」，看看能不能打出对应的词语。方便的话也可以把对应的dict.yaml传到github上，我可以帮忙看看。

**Comment by feyzhen** [comment, marked as answer]
我重新解压缩覆盖文件后重新部署，发现了是sublime的锅，默认打开dict.yaml的话，右下角缩进会自动把空格选上。只要把这个取消了再正常编辑词库就可以了
![图片](https://github.com/user-attachments/assets/301263ba-088d-4108-8f41-cd7602fbe299)
谢谢作者大大，嘿嘿。😘
